HadesAPI/router.go

```
@@ -167,13 +167,57 @@ func addBuildToQueue(c *gin.Context, producer hades.JobPublisher, statusPublishe
 		return
 	}
 
+	if p.QueuePayload.TimeoutSeconds < 0 {
+		slog.Error("Invalid timeout_seconds", "value", p.QueuePayload.TimeoutSeconds)
+		c.String(http.StatusBadRequest, "timeout_seconds cannot be negative")
+		return
+	}
+	if p.QueuePayload.TimeoutSeconds > payload.MaxTimeoutSeconds {
```

golangci-lint (2.12.2)

[error] QF1008: could remove embedded field "QueuePayload" from selector

(staticcheck)